msal4j-sdk/src/integrationtest/java/com.microsoft.aad.msal4j/AcquireTokenInteractiveIT.java

```diff
@@ -73,7 +72,45 @@ public void acquireTokenInteractive_ADFSv2_Federated(String environment) {
     public void acquireTokenInteractive_Ciam() {
         User user = labUserProvider.getCiamUser();
 
-        assertAcquireTokenCommon(user, TestConstants.CIAM_AUTHORITY, TestConstants.GRAPH_DEFAULT_SCOPE);
+        Map<String, String> extraQueryParameters = new HashMap<>();
+        extraQueryParameters.put("dc","ESTS-PUB-EUS-AZ1-FD000-TEST1");
+
+        PublicClientApplication pca;
+        try {
+            pca = PublicClientApplication.builder(
+                            user.getAppId()).
+                    authority("https://" + user.getLabName() + ".ciamlogin.com/")
+            .build();
+        } catch (MalformedURLException ex) {
+            throw new RuntimeException(ex.getMessage());
+        }
+
+        IAuthenticationResult result;
+        try {
+            URI url = new URI("http://localhost:8080");
+
+            SystemBrowserOptions browserOptions =
+                    SystemBrowserOptions
+                            .builder()
+                            .openBrowserAction(new SeleniumOpenBrowserAction(user, pca))
+                            .build();
+
+            InteractiveRequestParameters parameters = InteractiveRequestParameters
+                    .builder(url)
+                    .scopes(Collections.singleton(TestConstants.GRAPH_DEFAULT_SCOPE))
```

Comment:
Nit: Ideally, you shall test it with the scope returned by lab api, rather than using a hardcoded MS Graph scope. At one point in last quarter, other MSAL teams ran into issues with test with this. Perhaps the underlying CIAM service has been improved to support MS Graph since then, so we get away from it here.

Reply:
yeah, after our conversations, I tried it with different scopes, and this one worked just fine so thought of keeping it simple.